app.js

```diff
@@ -24,6 +25,12 @@ function generateCompMove() {
   return Math.floor(Math.random()*3);
 }
 
+function roundRoundup(){
+  alert('ROUND: ' + round + '\n' +
```

Comment:
One more thing, I would add in a message here saying what the computer picked and if it beat the player or not - I know you can ascertain that from how the scores change, but it's just a lot more intuitive for the user.